.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,26 @@
+name: 'CodeQL'
+
+on:
+  push:
+    branches: [ dev, master ]
+  pull_request:
+    branches: [ dev ]
+  schedule:
+    - cron: '0 10 * * *'
+
+jobs:
+  'Analyze':
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
```

Comment:
I guess we don't need a matrix for just one entry. Does it distinguish between js and ts btw?

Reply:
Their JavaScript setting includes TypeScript: https://help.semmle.com/QL/learn-ql/javascript/introduce-libraries-ts.html#overview

I added the matrix because it was given in the template from GitHub.